aws/resource_aws_resourcegroups_group_test.go

```diff
@@ -0,0 +1,115 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/resourcegroups"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSResourceGroup_basic(t *testing.T) {
+	resourceName := "aws_resourcegroups_group.test"
+	n := fmt.Sprintf("test-group-%d", acctest.RandInt())
+
+	desc1 := "Hello World"
+	desc2 := "Foo Bar"
+
+	query1 := `{
+  "ResourceTypeFilters": [
+    "AWS::EC2::Instance"
+  ],
+  "TagFilters": [
+    {
+      "Key": "Stage",
+      "Values": ["Test"]
+    }
+  ]
+}
+`
+
+	query2 := `{
+  "ResourceTypeFilters": [
+    "AWS::EC2::Instance"
+  ],
+  "TagFilters": [
+    {
+      "Key": "Hello",
+      "Values": ["World"]
+    }
+  ]
+}
+`
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
This `TestCase` should include a `CheckDestroy` function that verifies the removal of the resource from the API: https://www.terraform.io/docs/extend/testing/acceptance-tests/testcase.html#checkdestroy